tests/Doctrine/Tests/ORM/Functional/NativeQueryTest.php

```diff
@@ -370,11 +370,9 @@ public function testAbstractClassInSingleTableInheritanceSchemaWithRSMBuilderThr
         $rsm->addRootEntityFromClassMetadata(CompanyContract::class, 'c');
     }
 
-    /**
-     * @expectedException \InvalidArgumentException
-     */
     public function testRSMBuilderThrowsExceptionOnColumnConflict() : void
     {
+        $this->expectException('InvalidArgumentException');
```

Comment:
If we expect this exception to be thrown by `addRootEntityFromClassMetadata()`, this needs to be put before the method instead of the top of the test. Also:
```suggestion
        $this->expectException(\InvalidArgumentException::class);
```

plus `use`.